charts/prometheus-adapter/templates/deployment.yaml

```
@@ -10,6 +10,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   replicas: {{ .Values.replicas }}
+  strategy:
+{{ toYaml .Values.strategy | trim | indent 4 }}
```

nit: I prefer using nindent and the more succinct single line approach.
```suggestion
  strategy: {{ toYaml .Values.strategy | nindent 4 }}
```

---

Sure, it's definitely better approach. Thanks